src/main/java/reporting_system/BlockedUser.java

```diff
@@ -0,0 +1,50 @@
+package reporting_system;
+
+import data_access_storage.RequestModel;
+import data_access_storage.UserRepo;
+import data_access_storage.UserRepoManager;
+import entities.UserAccount;
+
+import javax.swing.*;
+import java.io.File;
+import java.io.IOException;
+import java.util.List;
+import java.util.Map;
+
```

Comment:
A comment regarding what this class is about would be helpful.  I'm a bit confused about what this class does